frontend/src/components/admin/AssignedStoryTranslationsTable.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable */
```

Comment:
I think this turns off eslint for the whole file, whats the specific line you want eslint to ignore?

Reply:
oops forgot to remove from before